_docs-v4/upgrading-from-v3.md

```diff
@@ -1060,9 +1060,9 @@ The following settings have been affected:
 <table>
 
 <tr>
-<th><a href='columnHeaderFormat'>columnHeaderFormat</a></th>
+<th><del>columnFormat</del></th>
```

Comment:
v3 actually also had `columnHeaderFormat`. `columnFormat` was a deprecated alias for it.

so, i think this should be two separate rows. one to say that the deprecated `columnFormat` has been removed and another that says the `columnHeaderFormat` accepts a new format